packages/v4/src/content/design-guidelines/content/HTTP-404-pages.md

```diff
@@ -0,0 +1,100 @@
+
+<img src="./img/404-page.png" alt="The PatternFly 404 page with added call outs to mark four main elements: Heading, next steps, suggested content, and a link to the home page." />
+
+1. **Heading:** Communicates what a 404 error is in plain language that users can understand. Avoid including “404” or “404 error” unless it’s clearly defined after: “404: That page no longer exists.”
+
+2. **Next steps:** Define how users can proceed, typically by inviting them to search the site, explore suggested content, or both.
+
+3. **Suggested content (optional):** Points users toward relevant pages such as onboarding information, reference guides, or FAQs.
+
+4. **Link to home page:** Provides a quick and easy way for users to navigate back to your site’s home page.
+
+## Writing 404 page content
+Consider your 404 page as a flight path instead of a dead end. Use clear, informative microcopy to point users in the right direction.
+
+**Use understandable language.** Skip technical jargon by writing your 404 error message in plain and specific terms.
+
+<div class="ws-content-table">
+
+| **Don't**                      | **Do**                  |
+|:-------------------------------:|:--------------------------:|
+| Error 404: Not found | 404: THat page no longer exists| 
+
+</div> 
+
+<br />
+
+**Avoid exclamations, colloquialisms, and excessive humor.** Write 404 error messages to be informative and repeatable. When users land on a page more than once, jokes grow stale. Steer clear of extraneous words like “Uh oh!” or “Oops!”.
+
+<div class="ws-content-table">
+
+| **Don't**                      | **Do**                  |
+|:-------------------------------:|:--------------------------:|
+| Uh oh, spaghetti-o! We lost that one | We lost that page | 
+| Oops! We dropped the ball | We couldn't find that page | 
+| Huh, that's odd... | That page no longer exists | 
+
+</div> 
+
+<br />
+
+**Avoid assigning blame to the user.** If your brand doesn’t use first-person plural (“we”) pronouns, use “that page” or “this page” as your subject instead.
+
+<div class="ws-content-table">
+
+| **Don't**                      | **Do**                  |
+|:-------------------------------:|:--------------------------:|
+| Your search came up empty | We can't find that page |
+| The page you're trying to reach doesn't exist | That page no longer exists |
+
+</div>
+
+<br />
+
+**Turn error into opportunity.** Always provide a link back to your site’s home page, and consider including supplemental microcopy to encourage users to explore options beyond just going back to where they came from.
+
+<div class="ws-content-table">
+
+| **Don't**                      | **Do**                  |
+|:-------------------------------:|:--------------------------:|
+| That page doesn't exist. | Another page might have what you need, so try searching PatternFly. |
```

Comment:
Also last thing - wondering why there are periods at the end of these examples, and not in any of the others

Reply:
I figured we should probably punctuate the examples in this table for two reasons:

1. That second line is punctuated on our actual 404 page we link to at the bottom of the page.
2. This table provides examples of next steps/body copy vs. the heading examples provided in previous ones. Typically, headings aren't punctuated, but next steps are. Should call out guidance for those punctuation best practices, maybe in the elements list? 

If so, we could then rephrase "supplemental microcopy" to "supplemental next steps" in the lines that introduce this table so that readers know they're reading an example of punctuated next step copy:

"**Turn error into opportunity.** Always provide a link back to your site’s home page, and consider including supplemental next steps to encourage users to explore options beyond just going back to where they came from."

What do you think?

Reply:
Ok that's what I figured! I think it just wasn't super clear to me which ones were heading suggestions and which ones were body suggestions, because the table right above this one seems like body suggestions, but the fact that they don't have periods make me think they're actually heading suggestions?

Reply:
Yes, the previous table contains heading suggestions! I think we can clarify this by mentioning whether the examples are headings or lines of body copy when we introduce them. I can tinker around with this to make sure we communicate that difference in a way that makes sense but also isn't too redundant. 👍

Reply:
@afialkoff , are you able to make the left side "Do" and the right side "Don't" ? That would align with our other guidelines in the style guide (example: https://www.patternfly.org/v4/ux-writing/brand-voice-and-tone).

Reply:
Absolutely, I'll update that now! 🙂 👍